training/model_management/components/import_model/spec.yaml

```diff
@@ -4,18 +4,33 @@ type: pipeline
 name: import_model
 display_name: Import model
 description: Import a model into a workspace or a registry
-version: 0.0.6
+version: 0.0.5
 
 # Pipeline inputs
 inputs:
   # pipeline specific compute
   compute:
     type: string
-    description: Common compute for model download, mlflow conversion and registration
+    optional: true
+    default: serverless
```

Comment:
> default: serverless [](http://example.com/codeflow?start=3&length=20)

Could we please check with Surya on the transition timelines?

Reply:
Confirmed from Surya , since it is in PuP there is no whitelisting needed